spec/facter/facts/linux/hypervisors/lxc_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+describe Facts::Linux::Hypervisors::Lxc do
+  describe '#call_the_resolver' do
+    subject(:fact) { Facts::Linux::Hypervisors::Lxc.new }
+
+    before do
+      allow(Facter::Resolvers::DockerLxc).to \
+        receive(:resolve).with(:hypervisor).and_return(hv)
+    end
+
+    context 'when resolver returns lxc' do
+      let(:hv) { { lxc: { 'name' => 'test_name' } } }
+      let(:value) { { 'name' => 'test_name' } }
+
+      it 'calls Facter::Resolvers::DockerLxc' do
+        fact.call_the_resolver
+        expect(Facter::Resolvers::DockerLxc).to have_received(:resolve).with(:hypervisor)
+      end
+
+      it 'returns virtual fact' do
+        expect(fact.call_the_resolver).to be_an_instance_of(Facter::ResolvedFact).and \
+          have_attributes(name: 'hypervisors.lxc', value: value)
+      end
+    end
+
+    context 'when resolver returns docker' do
+      let(:hv) { { docker: { 'id' => 'testid' } } }
+
+      it 'returns virtual fact' do
```

Comment:
test name should be 'returns nil' or 'returns nil fact'

Reply:
Done